python/tvm/autotvm/task/space.py

```diff
@@ -944,11 +939,10 @@ def from_json_dict(json_dict):
                 raise RuntimeError("Invalid config knob type: " + knob_type)
             entity_map[str(key)] = entity
 
-        return ConfigEntity(index, code_hash, template_key, entity_map, constraints)
+        return ConfigEntity(index, code_hash, entity_map, constraints)
 
     def __repr__(self):
-        return "%s,%s,%s,%d" % (str(self._entity_map)[12:-1], self.template_key,
-                                self.code_hash, self.index)
+        return "%s,%s,%d" % (str(self._entity_map)[12:-1], self.code_hash, self.index)
```

Comment:
Will this change our autotvm log format?

Reply:
Yes. Anyway this PR changes the workload name. So old tuning logs can no longer be used.

Reply:
Then we may need to bump the log format version, if we cannot ensure the compatibility, we should raise error when read the old version. Also, we can take this opportunity to fix our previous design, like changing the shortcut `i`, `r`, `v` to the full word: https://github.com/apache/incubator-tvm/blob/master/python/tvm/autotvm/record.py#L91

Reply:
Sure. Do you have any proposal for the change?